identity_presentation/src/commonMain/kotlin/net/thechance/mena/identity/presentation/components/AddressTypeSection.kt

```diff
@@ -0,0 +1,133 @@
+package net.thechance.mena.identity.presentation.components
+
+import androidx.compose.animation.animateColorAsState
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.unit.dp
+import mena.identity_presentation.generated.resources.Res
+import mena.identity_presentation.generated.resources.home
+import mena.identity_presentation.generated.resources.ic_home
+import mena.identity_presentation.generated.resources.ic_marker
+import mena.identity_presentation.generated.resources.ic_office
+import mena.identity_presentation.generated.resources.office
+import mena.identity_presentation.generated.resources.other
+import mena.identity_presentation.generated.resources.type
+import net.thechance.mena.designsystem.presentation.component.icon.Icon
+import net.thechance.mena.designsystem.presentation.component.text.Text
+import net.thechance.mena.designsystem.presentation.theme.theme.MenaTheme
+import net.thechance.mena.designsystem.presentation.theme.theme.Theme
+import net.thechance.mena.identity.presentation.screen.addresses.AddressType
+import org.jetbrains.compose.resources.painterResource
+import org.jetbrains.compose.resources.stringResource
+import org.jetbrains.compose.ui.tooling.preview.Preview
+
+@Composable
+fun AddressTypeSection(
+    selectedAddressType: AddressType?,
+    onClickAddressType: (AddressType) -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Column(modifier) {
+
+        Text(
+            text = stringResource(Res.string.type),
+            style = Theme.typography.title.small,
+            color = Theme.colorScheme.shadePrimary,
+            modifier = Modifier.padding(top = 12.dp, bottom = 4.dp)
```

Comment:
A tiny update here, also you can take a look on code in case you forgot to use the spacing
```suggestion
            modifier = Modifier.padding(top = Theme.spacing._12, bottom = Theme.spacing._4)
```